ColorPair_get_data.py

```diff
@@ -0,0 +1,26 @@
+from ColorPair import MAJOR_COLORS
+from ColorPair import MINOR_COLORS
+
+def color_pair_to_string(major_color, minor_color):
```

Comment:
`color_pair_to_string` is present in this file as well as in ColorPair.py?

Reply:
Removed it in the other file